packages/app-store/hubspot/lib/CrmService.ts

```diff
@@ -50,35 +53,40 @@ export default class HubspotCalendarService implements CRM {
     }`;
   };
 
-  private hubspotCreateMeeting = async (event: CalendarEvent) => {
-    const simplePublicObjectInput: SimplePublicObjectInput = {
-      properties: {
-        hs_timestamp: Date.now().toString(),
-        hs_meeting_title: event.title,
-        hs_meeting_body: this.getHubspotMeetingBody(event),
-        hs_meeting_location: getLocation(event),
-        hs_meeting_start_time: new Date(event.startTime).toISOString(),
-        hs_meeting_end_time: new Date(event.endTime).toISOString(),
-        hs_meeting_outcome: "SCHEDULED",
-      },
-    };
-
-    return hubspotClient.crm.objects.meetings.basicApi.create(simplePublicObjectInput);
-  };
+  private hubspotCreateMeeting = async (event: CalendarEvent, contacts: Contact[]) => {
+    try {
+      const simplePublicObjectInput: SimplePublicObjectInputForCreate = {
+        properties: {
+          hs_timestamp: Date.now().toString(),
+          hs_meeting_title: event.title,
+          hs_meeting_body: this.getHubspotMeetingBody(event),
+          hs_meeting_location: getLocation(event),
+          hs_meeting_start_time: new Date(event.startTime).toISOString(),
+          hs_meeting_end_time: new Date(event.endTime).toISOString(),
+          hs_meeting_outcome: "SCHEDULED",
+        },
+        associations: contacts.reduce((associations, contact) => {
```

Comment:
Instead of making a separate call to associate a meeting with a contact, we can do it in one call. It'll actually throw a type error if we don't.